test/e2e/configs.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration || interactive
```

Comment:
I would still remove this line (as mentioned [here](https://github.com/observatorium/api/pull/246#discussion_r826815823))